packages/api/src/aggregate/WhereClause.ts

```diff
@@ -50,7 +50,7 @@ type MakeFilter<K extends PossibleWhereClauseFilters, V> = K extends string ? {
 type BaseFilter<T> =
   | T
   | MakeFilter<"$eq" | "$ne", T>
-  | MakeFilter<"$in", Array<T>>
+  | MakeFilter<"$in", ReadonlyArray<T>>
```

Comment:
Can we have a tiny test for this? Its the type of thing that would be easy to regress in a year or two with refactorings.